src/main/resources/archetype-resources/Jenkinsfile

```diff
@@ -1,14 +1,20 @@
 buildMvn {
-  publishModDescriptor = 'no'
-  runLintRamlCop = 'no'
-  publishAPI = 'no'
-  mvnDeploy = 'no'
+  publishModDescriptor = false
+  mvnDeploy = false
+  publishAPI = false // Deprecated. See doApiDoc.
+  runLintRamlCop = false // Deprecated. See doApiLint.
```

Comment:
I reckon just remove these two deprecated properties. It still seems to confuse some people.

(I will continue adjustment of the dev.f.o docs, which attempt to get projects to transition to the new tools.)

Reply:
Thanks, done!